pkg/controller/migplan/request_mapfuncs.go

```diff
@@ -0,0 +1,47 @@
+/*
+Copyright 2019 Red Hat Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package migplan
```

Comment:
Okay.  The name _request_mapfuncs_ throw me for a while.  These are mapping functions passed to the handler when adding the watch.  Good that they're carved out into a separate module.  I wonder if naming the module based on the subject or concern would be more helpful.  Perhaps a `handler.go` module so reader would know it contains watch _handler_ related things?

Reply:
ACK, works for me

Reply:
ACK, changed